### PR TITLE
New torrents and https URLs for Telia firmwares

### PR DESCRIPTION
**Pull request by wuseman**
Added torrent and http url for vdnt-o firmware.

### CODE REVIEW

**Comment by wuseman** [comment]
Added latest source for telias rbi files, they are available for download on any ip.

**Comment by LuKePicci** [comment]
Almost fine, I also need torrents for the other RBI you linked, make sure to include the telia server links as web seeds.
Remove the "-XXXXXXX" suffix from the VDNT-O table.
Remove the * after any new HTTPS link from the new server as it is freely accessible from the whole internet (no more mgmt vlan is needed to reach them).

File: docs/Repository.md [hunk 1]
```diff
@@ -342,7 +346,9 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
 | 2      | 17.2.0339        | 2018-02-28 | #A            | [HTTP](http://131.116.22.230/172339h1441002closed.rbi)* - [Torrent](https://github.com/hack-technicolor/hack-technicolor/blob/master/torrents/vbnt-h/172339h1441002closed.rbi.torrent?raw=true) |
 | 2      | 17.2.0339        | 2018-04-16 | #A            | [HTTP](http://131.116.22.230/172339h1441018closed.rbi)* - [Torrent](https://github.com/hack-technicolor/hack-technicolor/blob/master/torrents/vbnt-h/172339h1441018closed.rbi.torrent?raw=true) |
 | 3      | 17.2.0405        | 2019-03-29 | -             | [HTTP](http://131.116.22.230/172405h1441028closed.rbi)* - [Torrent](https://github.com/hack-technicolor/hack-technicolor/blob/master/torrents/vbnt-h/172405h1441028closed.rbi.torrent?raw=true) |
+| 3      | 17.2.0405        | 2019-03-29 | -             | [HTTPS](https://rgw.teliacompany.com:7547/172405h1441028closed.rbi)* - [Torrent](https://github.com/hack-technicolor/hack-technicolor/blob/master/torrents/vbnt-h/172405h1441028closed.rbi.torrent?raw=true) |
```
Comment (LuKePicci) [inline docs/Repository.md:349]: No need to duplicate this line, just add HTTPS to the previous one and rebuild the torrent with new web seed.

**Review by wuseman — COMMENTED** [review]
Sure, I will fix the torrent files.

**Pull request merged**